packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -71,9 +94,30 @@ const mapDispatchToProps = {
   completePaymentRequest,
 }
 
+interface DisplayNameProps {
+  recipient: Recipient
+  recipientAddress?: string | null
+  t: TFunction
+}
+
+function getDisplayName({ recipient, recipientAddress, t }: DisplayNameProps) {
```

Comment:
Should this be in `recipient.ts`? Also, it looks very similar to the one in `Avatar.ts`, wondering if we can DRY those up

Reply:
Moved to `recipient.ts` and left a note for whoever redesigns `Avatar` that there is a function that they can import. Reluctant to put in the effort to make it extensible if I am not sure if/how it will be used once redesigned